test-projects/deadlines/worker.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 Adobe Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+'use strict'
+
+const { worker } = require('@adobe/asset-compute-sdk');
+const fs = require('fs').promises;
+
+exports.main = worker(async (source, rendition) => {
+    console.log("__OW_DEADLINE", process.env.__OW_DEADLINE);
+    console.log("__OW_ACTIVATION_ID", process.env.__OW_ACTIVATION_ID);
+
+    await fs.writeFile(rendition.path, JSON.stringify({
+        __OW_DEADLINE: process.env.__OW_DEADLINE,
+        __OW_ACTIVATION_ID: process.env.__OW_ACTIVATION_ID
+    }))
```

Comment:
Nitpick: Isn't a semicolon missing at the end of this line?

Reply:
Yeah, that's in the test-projects code which is ignored by eslint (since there are some things in there that make it fail, like deliberately "broken" workers).

https://github.com/adobe/aio-cli-plugin-asset-compute/blob/master/.eslintignore